ra/SAPHanaTopology

```diff
@@ -1131,7 +1124,8 @@ sidadm=""
 InstanceName=""
 InstanceNr=""
 DIR_EXECUTABLE=""
-SAPHanaFilter="ra-act-dec-lpa"
+SAPHanaFilter="RA-ACT-DEC-LPA"
+
```

Comment:
see comment above

Reply:
Thx for the review. 
There are couple of ways to have the filter specified lower|upper

- use tr, but this has the same drawbacks as grep
- use variable substitution provided by bash ${var^^} ; unfortunately this is only provided since bash4 (SLES11.4 and RHEL6 are still shipped with bash3)
- use the builtin declare -u (this will make sure variable content is always uppercase)
[test_super_ocf_log_new_with_lowercaseFilter.sh.txt]

I tested the 3rd open successfully on bash3
(https://github.com/SUSE/SAPHanaSR/files/9578707/test_super_ocf_log_new_with_lowercaseFilter.sh.txt)